pom.xml

```diff
@@ -82,8 +82,8 @@
         <freemarker.version>2.3.31</freemarker.version>
         <hikari.version>3.4.5</hikari.version>
         <immutables.version>2.9.2</immutables.version>
-        <jackson.version>2.13.3</jackson.version>
-        <jackson-jsr310.version>2.13.3</jackson-jsr310.version>
+        <jackson.version>2.14.0-rc2</jackson.version>
+        <jackson-jsr310.version>2.14.0-rc2</jackson-jsr310.version>
```

Comment:
not 100% sure about listing RC builds.  It's fine for dev, for prod I'm not so sure.